draft-wang-ppm-differential-privacy.md

```diff
@@ -451,9 +465,81 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
+The binary randomized response mechanism specified here aims to provide the
+DP guarantee at the Client level. It is the same as the one specified in
+Appendix C.1 of {{MJTB+22}}.
+
+To apply noise at Client side, the Client flips a coin at each coordinate of its
+vector. If the coin turns out to be a head, the Client will change the value `x`
+at that coordinate to `1 - x`. The probability of the coin giving a head is
+`1 / (e^(EPSILON_0) + 1)`, where `EPSILON_0` provides the pure `EPSILON_0`-DP
+guarantee, in the deletion-DP model (Definition II.4 of {{EFMR+20}}), which
```

Comment:
1. If you say the words "DP guarantee", then you probably aren't being clear enough. Do you mean that we get `EPSILON_0`-DP in the OC trust model?
2. In the appendix we say that DAP is not compatible with deletion-DP. Indeed, the way we define neighboring databases in the section above rules this out.

Reply:
for 1. I think it's ok to see EPSILON_0-DP guarantee, since this is client level DP noise, it doesn't really to do with the DP guarantee for the whole aggregation, which then relates to the trust models.